test/typescript/model/Ark.ts

```diff
@@ -1,4 +1,4 @@
-import {Supertype, supertypeClass, property} from '../../../index';
+import {Supertype, supertypeClass, property} from '../../../dist/index';
```

Comment:
why importing from "dist" directory?

Reply:
This is for compilation purposes for the tests only.  The dist/index is the entry point of our application (and what gets published) so it makes sense for our tests for supertype to import and use supertype the same way our modules do.